src/components/MoveList.js

```diff
@@ -24,9 +24,17 @@ listTemplate.innerHTML = String.raw`
     }
 
     .btns-container{
+      display: flex;
+      justify-content: center;
+    }
```

Comment:
this works. Personally, I would rather have you do this all in grid for consistency and learning purposes